scripts/diffThemes.test.ts

```diff
@@ -0,0 +1,34 @@
+import {diffThemes} from './diffThemes'
+
+describe('diffThemes', () => {
+  const themesArray: [string, string[]][] = [
+    ['light', ['color', 'color.text']],
+    ['dark', ['color', 'color.text']],
+  ]
+
+  it('Throws error if mainTheme name does not exist', () => {
+    expect(() => diffThemes('invalid', themesArray)).toThrow()
+  })
+
+  it('Outputs message for missing tokens', () => {
+    const withMissing = JSON.parse(JSON.stringify(themesArray))
```

Comment:
I'm possibly missing something fairly obvious, but do you need to stringify and reparse this array? What does it accomplish?

Reply:
> Are you trying to avoid mutating the original array?

Yes, and since it is nested a normal spread does not do it.